unstake.js

```diff
@@ -111,6 +111,26 @@ const createRedeemRequest = async () => {
 }
 // createRedeemRequest();
 
+
+const checkRedeemEvent = async () => {
+    //insert secret here below. Replace Goerli with any other
+    const provider = new ethers.providers.WebSocketProvider(
+        `wss://goerli.infura.io/ws/v3/${INFURA_SECRET}`, "goerli"
+      );
+    
+    const redeemManagerContract = new ethers.Contract(CONTRACT_ADDRESS, ABI, provider.getSigner());
+
+    redeemManagerContract.on("RequestedRedeem",(owner, height, amount, maxRedeemableEth, id) => {
+      console.log('RequestedRedeem event');
+      console.log(`Owner of redeem ${owner}`)
+      console.log(`Height ${height.toString()}`);
+      console.log(`Amount of LsETH to redeem ${amount.toString()}`);
+      console.log(`Maximum amount of ETH to redeem ${maxRedeemableEth.toString()}`);
+      console.log(`Request Redeem ID ${id}`);
+    });
+}
+// checkRedeemEvent
```

Comment:
Might consider adding the websocket code block to a separate .js (maybe websocket.js) file, that way a user can run the websocket.js and then run the unstake.js functions